docker/Dockerfile

```diff
@@ -12,13 +12,25 @@ FROM --platform=$BUILDPLATFORM ghcr.io/restatedev/dev-tools:latest AS planner
 COPY . .
 RUN just chef-prepare
 
-FROM --platform=$BUILDPLATFORM ghcr.io/restatedev/dev-tools:latest AS builder
+FROM --platform=$BUILDPLATFORM ghcr.io/restatedev/dev-tools:latest AS base
 ARG TARGETARCH
 # Overrides the behaviour of the release profile re including debug symbols, which in our repo is not to include them.
 # Should be set to 'false' or 'true'. See https://doc.rust-lang.org/cargo/reference/environment-variables.html
 ARG CARGO_PROFILE_RELEASE_DEBUG=false
+ARG ACTIONS_CACHE_URL=''
+ARG ACTIONS_RUNTIME_TOKEN=''
+ARG SCCACHE_GHA_ENABLED=''
```

Comment:
Shall we link https://github.com/mozilla/sccache/blob/main/docs/GHA.md as reference to why these args are needed?